pkgs/unified_analytics/lib/src/analytics.dart

```diff
@@ -760,6 +800,12 @@ class FakeAnalytics extends AnalyticsImpl {
     // for internal methods in the `Analytics` instance
     sentEvents.add(event);
   }
+
+  @override
+  void _sendError(Event event) {
+    super._sendError(event);
```

Comment:
why do we call super here?

Reply:
We call super here because the difference in logic for the `_sendError` method in the `Analytics` and `FakeAnalytics` instance is that the fake instance adds it to the `sentEvents` list.  This list is checked in our tests to confirm that an error event was sent

Reply:
ahh, I see. this makes sense.